src/controller/editingcontroller.js

```diff
@@ -23,6 +23,7 @@ import {
 } from '../conversion/model-selection-to-view-converters';
 
 import EmitterMixin from '@ckeditor/ckeditor5-utils/src/emittermixin';
+import '../../theme/theme.scss';
```

Comment:
That's a good point, actually. If we import things directly, switching to another theme will require changes in each single js file that imports SCSS vs. changing theme.scss file only. I wonder whether this could be possible.